File: beginner/c/1154.c
#include <stdio.h>
int main(){

    int i, idade, soma = 0, total = 0;
    double media;

    for(;;){
        scanf("%d", &idade);
        if(idade >= 0){
            soma += idade;
            total++;
        }else{
            break;
        }
    }

    media = (double) soma / total;

    printf("%.2lf\n", media);

    return 0;
}

File: beginner/javascript/1154.js
var input = require('fs').readFileSync('stdin', 'utf8');
var valores = input.split('\n');

var idade, soma = 0, total = 0, media;

for(;;){
    idade = parseInt(valores.shift());
    if(idade >= 0){
        soma += idade;
        total++;
    }else{
        break;
    }
}

media = soma / total;

console.log(`${media.toFixed(2)}`);